src/modules/LanItems.rb

```diff
@@ -2460,6 +2388,173 @@ def drop_hosts(ip)
       Host.set_names(ip, [])
     end
 
+    # Exports udev rules for AY profile
+    def export_udevs(devices)
+      devices = deep_copy(devices)
+      ay = { "s390-devices" => {}, "net-udev" => {} }
+      if Arch.s390
```

Comment:
This branch is not covered with unit tests. And I suspect that our integration tests do not test s390 so we should write unit tests for it.